tests/components/alexa/test_flash_briefings.py

```diff
@@ -75,6 +79,30 @@ async def test_flash_briefing_invalid_id(alexa_client):
     assert text == ""
 
 
+async def test_flash_briefing_no_password(alexa_client):
+    """Test for no Flash Briefing password."""
+    req = await _flash_briefing_req(alexa_client, "weather", password=None)
+    assert req.status == HTTP_UNAUTHORIZED
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5745133)